test/AuthTokenVerifier.test.ts

```diff
@@ -119,16 +118,46 @@ describe("AuthTokenVerifier", () => {
 
     describe("With Ethers signature", () => {
       beforeAll(() => {
+        delegate = Wallet.createRandom();
         createClaim = createEthersClaim;
       });
       delegateTests();
     });
 
     describe("With Json web signature", () => {
       beforeAll(() => {
+        delegate = Wallet.createRandom();
         createClaim = createJwtClaim;
       });
       delegateTests();
     });
+
+    describe("With P256 signature", () => {
+      beforeAll(() => {
+        const ecKey = ECKey.createECKey('prime256v1');
+        const publicKey = `0x${ecKey.publicCodePoint.toString('hex')}`;
+        delegate = {
+          publicKey,
+          address: publicKey,
+        };
+        createClaim = () => jsonwebtoken.sign(payload, ecKey.toString('pem'), { algorithm: 'ES256', noTimestamp: true });
+      });
+
+      // it("sigAuth delegate should be authenticated", async () => {
+      //   const document = mockDocument(identity);
+      //   document.publicKey.push({
+      //     id: `did:${Methods.Erc1056}:${delegate.address}#${PubKeyType.SignatureAuthentication2018}`,
+      //     type: PubKeyType.SignatureAuthentication2018,
+      //     publicKeyHex: delegate.publicKey,
+      //   } as IPublicKey);
+      //   verifier = new AuthTokenVerifier(document);
+      //   claim = await createClaim(delegate);
+      //   const did = await verifier.verify(claim);
+
+      //   assert.strictEqual(did, document.id);
+      // });
```

Comment:
Is this intended to be commented?
```suggestion
      // it("sigAuth delegate should be authenticated", async () => {
      //   const document = mockDocument(identity);
      //   document.publicKey.push({
      //     id: `did:${Methods.Erc1056}:${delegate.address}#${PubKeyType.SignatureAuthentication2018}`,
      //     type: PubKeyType.SignatureAuthentication2018,
      //     publicKeyHex: delegate.publicKey,
      //   } as IPublicKey);
      //   verifier = new AuthTokenVerifier(document);
      //   claim = await createClaim(delegate);
      //   const did = await verifier.verify(claim);

      //   assert.strictEqual(did, document.id);
      // });
```

Reply:
Sorry, it is left after debugging

Reply:
No problem. Please let me know when to review again